third_party/2and3/yaml/cyaml.pyi

```diff
@@ -18,3 +20,25 @@ class CLoader(CParser, SafeConstructor, Resolver):
 
 class CSafeLoader(CParser, SafeConstructor, Resolver):
     def __init__(self, stream: Union[str, bytes, _Readable]) -> None: ...
+
+class CDangerLoader(CParser, Constructor, Resolver): ...  # undocumented
+
+class CEmitter:
+    def __init__(self, stream: IO[Any], canonical: Any = ..., indent: Optional[int] = ...,
+                 width: Optional[int] = ..., allow_unicode: Any = ..., line_break: Optional[str] = ...,
+                 encoding: Optional[str] = ..., explicit_start: Any = ..., explicit_end: Any = ...,
```

Comment:
Looking at https://github.com/yaml/pyyaml/blob/a9c28e0b521967f5330f0316edd90a57f99cdd32/ext/_yaml.pyx#L1246 and https://github.com/yaml/pyyaml/blob/a9c28e0b521967f5330f0316edd90a57f99cdd32/ext/_yaml.pyx#L1319, it seems like the types of `encoding` and the keys and values in `tags` should all be `Text` instead of `str` because bytes and unicode are both handled.